### PR TITLE
Feat: Added typography (text utilities)

### PR DESCRIPTION
**Pull request by GauravDe-git**
image
image

### CODE REVIEW

**Review by Divya70 — APPROVED** [review]
good to merge

File: Pages/Avatar/avatar.html [hunk 1]
```diff
@@ -51,6 +51,9 @@ <h4 class="nav-page-heading">Components</h4>
                 <li class="nav-page-li">
                     <a href="/Pages/Navigation/navigation.html">Navigation</a>
                 </li>
+                <li class="nav-page-li">
+                    <a href="/Pages/Text-utilities/textUtilities.html">TextUtils</a>
+                </li>
                 <li class="nav-page-li">
                     <a href="">Modal</a>
```
Comment (VishalPatil18) [inline Pages/Avatar/avatar.html:58]: You could give value to the empty `href` attribute

File: Pages/Text-utilities/textUtilities.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    font-weight: 300;
+}
+
+.text-color-grey{
+    color: grey;
+}
+
+.text-center{
+    text-align: center;
+}
```
Comment (VishalPatil18) [inline Pages/Text-utilities/textUtilities.css:49]: You could add an empty line at the end of the file to avoid such warnings from Github.

**Review by VishalPatil18 — APPROVED** [review]
Just a couple of suggestions. The rest looks quite good. Great work @GauravDe-git 👍

**Pull request merged**